src/views/Dog/Dog.js

```diff
@@ -22,10 +22,16 @@ export default function Dog() {
   if (loading) {
     return <h3>Loading</h3>;
   }
+  const handleDelete = async (e) => {
+    e.preventDefault();
+    await deleteDog(dog.id);
```

Comment:
Would advise using a try/catch for delete too. Basically, any time you hit your database.